src/chronometer.js

```diff
@@ -1,33 +1,50 @@
+
+
 class Chronometer {
   constructor() {
-    // ... your code goes here
+this.currentTime = 0;
+this.intervalId = null
   }
 
   start(printTimeCallback) {
-    // ... your code goes here
-  }
+//use an arrow function 
+
+this.intervalId = setInterval(() => {
+  this.currentTime++ ;
+  if (typeof printTimeCallback === 'function'){
+    printTimeCallback()}
+}, 1000);
+}
 
   getMinutes() {
-    // ... your code goes here
-  }
+    return Math.floor(this.currentTime / 60)  ; 
+    }
 
   getSeconds() {
-    // ... your code goes here
+    return this.currentTime % 60
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
-  }
+       
+    if (value < 10){
+       return '0' + value.toString();
+     } else  {
+      return value.toString();
+     } 
+     }   
 
   stop() {
-    // ... your code goes here
+    clearInterval(this.intervalId)
   }
 
-  reset() {
-    // ... your code goes here
+  reset() { 
+    this.currentTime = 0
   }
 
   split() {
-    // ... your code goes here
+    this.getMinutes()
+    this.getSeconds()
+    this.computeTwoDigitNumber()
```

Comment:
That method need a value to work, so you could pass the `this.getSeconds()` or `this.getMinutes()` to provide the value and turn it into a string and use it later